internal/scheduler/jobdb/job.go

```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/armadaproject/armada/internal/scheduler/internaltypes"
```

Comment:
This should be with the other armada imports (it's bit sad that goimports doesn't understand this!)

Reply:
done